bratislava/olo.sk
# 948 wastecollectiondays next

Add functionality to WasteCollectionDays Tables
Menu -> Odpad -> Zisti si svoj odvozovy den

Previously header of the tables in WasteCollectionDays was set for one waste type, with this change also different headers are able to be set for different tables for the same waste type. These different tables are displayed in tabs. Therefore new content type item **WasteCollectionDaysTab** was set with information about the table. This content type is added to **WasteCollectionDaysSection**

How to test:
Add dynamic section "Odvozové dni" in that section **ignore** setup of waste collection type, header of visible columns (this will be removed later on)
Add in this section "Záložka s tabuľkou" and fill in required fields (assuming you have production dump applied with some items in collection type "Odvozove dni")

If only one tab is added it should look like this:
![image](https://github.com/user-attachments/assets/42912c6a-5050-483e-b634-d581bb1dcfff)

If more tabs are added, those are clicable (usually only 2)
![image](https://github.com/user-attachments/assets/f7fa609c-451f-404b-84d5-d241c56e9801)
